dev/AppNotifications/AppNotificationBuilder/AppNotificationBuilder.idl

```diff
@@ -99,12 +99,22 @@ namespace Microsoft.Windows.AppNotifications.Builder
         AppNotificationProgressBar BindValueStringOverride();
     };
 
+    [contract(AppNotificationBuilderContract, 2)]
+    struct AppNotificationComboBoxItem
+    {
+        String Id;
+        String Content;
+    };
+
     [contract(AppNotificationBuilderContract, 1)]
     runtimeclass AppNotificationComboBox
     {
         AppNotificationComboBox(String id);
 
+        [deprecated("Use ItemList instead of Items. For more info, see MSDN.", deprecate, AppNotificationBuilderContract, 2)]
         Windows.Foundation.Collections.IMap<String, String> Items;
```

Comment:
The Items property has been deprecated and replaced by the itemList property (couldn't think of a better name, but open to suggestions).

In this implementation, AppNotificationComboBox does the conversion between this old property and its internal representation. The alternative would be to throw a "not implemented" error. Throwing is easier but I'm not sure this would be appropriate behavior. @MikeHillberg do you have an opinion on this?